tests/integration/components/flexberry-groupedit-test.js

```diff
@@ -20,12 +24,121 @@ moduleForComponent('flexberry-groupedit', 'Integration | Component | Flexberry g
     UserSettingsService.reopen({
       isUserSettingsServiceEnabled: false
     });
+
+    // Override base component's reference to current controller.
+    editFormController = EditFormController.create(App.__container__.ownerInjection());
+    FlexberryBaseComponent.prototype.currentController = editFormController;
   },
+
   afterEach: function() {
+    // Restore base component's reference to current controller to its initial state.
+    FlexberryBaseComponent.prototype.currentController = null;
+
     Ember.run(App, 'destroy');
   }
 });
 
+test('ember-grupedit cellComponent test', function(assert) {
+  let store = App.__container__.lookup('service:store');
+
+  Ember.run(() => {
+    let model = store.createRecord('components-examples/flexberry-groupedit/shared/aggregator');
+    let testComponentName = 'my-test-component-to-count-rerender';
+    let tempCellComponent = {
+      componentName: 'flexberry-checkbox',
+      componentProperties: null
+    };
+
+    // Remove getCellComponent method from current controller's instance to force groupedit use componennt defined in its cellComponent property.
+    editFormController.set('getCellComponent', null);
```

Comment:
This does not make sense, because in integration tests, components are rendered without a controller.
It is necessary to removing these and related changes.